code/modules/admin/topic.dm

```diff
@@ -1395,19 +1395,33 @@
 			to_chat(usr, "This can only be used on instances of type /mob/living/carbon/human.")
 			return
 
-		var/obj/item/reagent_containers/food/snacks/cookie/cookie = new(H)
-		if(H.put_in_hands(cookie))
+		//if(is_species(H, /datum/species/lizard))
+		
+		//let's keep it simple
+		//milk to plasmemes and skeletons, meat to lizards, electricity bars to ethereals, cookies to everyone else
+		//var/obj/item/reagent_containers/food/snacks/cookie/cookie = new(H)
+		var/cookiealt = /obj/item/reagent_containers/food/snacks/cookie
+		if(is_species(H, /datum/species/skeleton))
+			cookiealt = /obj/item/reagent_containers/food/condiment/milk
+		if(is_species(H, /datum/species/plasmaman))
+			cookiealt = /obj/item/reagent_containers/food/condiment/milk
+		if(is_species(H, /datum/species/ethereal))
+			cookiealt = /obj/item/reagent_containers/food/snacks/energybar
+		if(is_species(H, /datum/species/lizard))
+			cookiealt = /obj/item/reagent_containers/food/snacks/meat/slab
+		var/obj/item/new_item = new cookiealt(H)
+		if(H.put_in_hands(new_item))
 			H.update_inv_hands()
 		else
-			qdel(cookie)
-			log_admin("[key_name(H)] has their hands full, so they did not receive their cookie, spawned by [key_name(src.owner)].")
-			message_admins("[key_name(H)] has their hands full, so they did not receive their cookie, spawned by [key_name(src.owner)].")
+			qdel(new_item)
+			log_admin("[key_name(H)] has their hands full, so they did not receive their [new_item], spawned by [key_name(src.owner)].")
+			message_admins("[key_name(H)] has their hands full, so they did not receive their [new_item], spawned by [key_name(src.owner)].")

```

Comment:
Not sure if text will get formated correctly. This might need to be ``new_item.name`` instead of just ``new_item``

Reply:
it worked when i tested it